app/views/idv/phone/new.html.erb

```diff
@@ -38,7 +38,14 @@
 </ul>
 
 <p>
-  <%= t('idv.messages.phone.final_note_html') %>
+    <a href="<% help_center_redirect_url(
```

Comment:
nit: I think I would expect this to be a `link_to`

Reply:
Good call, thanks updated in 211f1dda9